### DISCUSSION TITLE
Shared presets files lookup

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#23724] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

One org is used for the renovate runner and the global shared presets for other github.com orgs.
I'd like that some renovated orgs having their own org shared presets tracked in org/.github repository.
Renovate consumes org/.github presets if it exists. No need to specify it explicitly.
The reason for that is that not all renovated orgs have this their own org shared preset file. And the global shared preset is consumed by repos in other platforms.

### What you tried so far.

The current config.js for each self-hosted platform runner:

```
  onboardingConfig: {
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
      "github><global org>/.github"
    ]
  },
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by lmilbaum** [comment, marked as answer]
bump

**Reply by viceice** [reply]
I don't really understand what the goal is. 🤷‍♂️

**Reply by lmilbaum** [reply]
I think that the [following](https://github.com/renovatebot/renovate/blob/6c42022f78c9ff73983e95c97099bb0cc4442b27/docs/usage/config-presets.md?plain=1#L219) solves my ask

**Reply by viceice** [reply]
presets can extend other presets, yes

**Reply by lmilbaum** [reply]
Thanks. I've used that solution but I still needed to update ~10 repos because they are already onboarded with the onboardingConfig.

**Reply by viceice** [reply]
yes, that's still a manual task todo 🙃
